### PR TITLE
Code Maintenance Housekeeping

### PR DESCRIPTION
**Pull request by robweber**
As we get close to merging in some of the code changes I thought we could get this into `master` first if that's ok. Basically some housekeeping stuff. 

__Issue Templates__
Added some basic templates for bugs, features, and informational requests. Should help with some of the Issues going forward. The informational ones I added mainly due to all the Issues that are just links to other forks or similar projects. We could go ahead and start posting those to like "Links" page on the wiki or something? 

__Contributing__
I went through a bunch of contributing files and pieced together something I thought made sense. With multiple author changes being merged in I think it's fair there are no surprises to people on the process for things. It's also just something we can stand behind going forward. 

__License__
I know this is in one of the other PRs but again, I think it's important this exist prior to merging in other stuff. No surprises regarding how this code is being maintained when contributing. 

This has no code modifications so I think we could merge this now. If there are any wording changes let me know.

### CODE REVIEW

**Review by qubist — CHANGES_REQUESTED** [review]
Looks great, left a few small suggestions for tweaks (I found that I couldn't make the changes myself, possibly because it's a PR from @robweber's fork?)

One other parallel change is that the new Contributing section of the README.md should link to this new contributing document. We can make sure that lines up later when merging #37

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
```
[the remark this answers is not in the document]
Comment (robweber) [reply .github/ISSUE_TEMPLATE/feature_request.md:3]: That's kind of funny as there were Github generated files from a tool. I made them consistent.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -32,14 +32,14 @@ We use GitHub issues to track issues. Report a something by [opening a new issue
 - What actually happens
 - Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
 
-A thorough bug reports is your best change things can be fixed quickly. 
+A thorough bug report is your best change things can be fixed quickly.
```
Comment (qubist) [inline .github/CONTRIBUTING.md:35]: I still see one lil typo here: change -> chance. After that I'm good for a merge!

**Review by qubist — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (qubist) [reply .github/CONTRIBUTING.md:35]: Sorry I didn't make that clearer in my comment before 🤖

**Review by qubist — APPROVED** [review]
Let's merge 🚀

**Pull request merged**